tools/spelling/spelling_dictionary.txt

```diff
@@ -1458,3 +1458,4 @@ SSLKEYLOGFILE
 DLB
 PCIE
 EDNS
+cname
```

Comment:
nit: can we use `CNAME` for the comments?

Reply:
ack, done